apps/fluent-tester/package.json

```diff
@@ -36,7 +36,7 @@
     "react": "16.11.0",
     "react-native": "0.62.2",
     "react-native-svg-transformer": "^0.14.3",
-    "react-native-windows": "^0.0.0-master.45",
+    "react-native-windows": "0.0.0-master.47",
```

Comment:
Do you need to get rid of the ^? Do you need to pin to this version because of some other config?

Reply:
Per request from @FalseLobster. I believe we do, not entirely sure.

Reply:
I would have thought a simple yarn upgrade would have picked up the right things

Reply:
but maybe the tags make things weird

Reply:
From Alicia: "the caret range is kind of a lie, as it is, deleting the yarn.lock file (which I'm assuming is how you "broke" the lock file in the first place) shouldn't be a problem, but we apparently have a hard dependency on a specific react-native-windows version that isn't being communicated by our use of the caret range"